package.json

```diff
@@ -33,7 +33,6 @@
     "npm": "^8"
   },
   "jest": {
-    "preset": "ts-jest",
```

Comment:
don't think this is needed any more?

Reply:
It's a base config? We are pointing at the `ts-jest` module? https://jestjs.io/docs/configuration#preset-string

Reply:
I spent ages reading the docs and I still don't know. The tests run successfully without it